src/pages/en/guides/images.md

```diff
@@ -243,21 +244,43 @@ export const galaxy = 'https://astro.build/assets/galaxy.jpg';
 <Picture src={galaxy} widths={[200, 400, 800]} aspectRatio={16/9} sizes="(max-width: 800px) 100vw, 800px" alt="Outer space." />
 ```
 
-## Alt Text
+### Setting Default Values
 
-Not all users can see images in the same way, so accessibility is an especially important concern when using images. Use the `alt` attribute to provide [descriptive alt text](https://www.w3.org/WAI/tutorials/images/) for images. 
+Currently, there is no way to specify default values for all `<Image />` and `<Picture />` components. Required attributes should be set on each individual component.
 
-This attribute is required for the Image integration's `<Image />` and `<Picture />` components. These components will throw an error if no alt text is provided. 
+As an alternative, you can wrap these components in another Astro component for reuse. For example, you could create a component for your blog post images:
 
-If the image is merely decorative (i.e. doesn’t contribute to the understanding of the page), set `alt=""` so that the image is properly understood and ignored by screen readers.
+```astro title="src/components/BlogPostImage.astro"
+---
+import { Picture } from '@astrojs/image/components';
 
+const {src, ...attrs} = Astro.props;
+---
+<Picture src={src} widths={[400, 800, 1500]} sizes="(max-width: 767px) 100vw, 736px" {...attrs} />
+
+<style>
+  img, picture :global(img), svg {
+    @apply rounded-xl shadow-lg my-10;
```

Comment:
Torn on using Tailwind syntax here. Do we hate a slightly longer, vanilla CSS version?

```suggestion
    margin-block: 2.5rem;
    border-radius: 0.75rem;
```

Reply:
Oh not torn at all! I explicitly wrote in my comment at the beginning, PLZ UN-TAILWIND the example!  😅